prijateli_tree/settings.py

```diff
@@ -0,0 +1,40 @@
+from os import environ
+
+
+SESSION_CONFIGS = [
+    # dict(
+    #     name='public_goods',
+    #     app_sequence=['public_goods'],
+    #     num_demo_participants=3,
+    # ),
+]
+
+# if you set a property in SESSION_CONFIG_DEFAULTS, it will be inherited by all configs
+# in SESSION_CONFIGS, except those that explicitly override it.
+# the session config can be accessed from methods in your apps as self.session.config,
+# e.g. self.session.config['participation_fee']
+
+SESSION_CONFIG_DEFAULTS = {
+    "real_world_currency_per_point": 1.00,
+    "participation_fee": 0.00,
+    "doc": "",
+}
+
+PARTICIPANT_FIELDS = []
+SESSION_FIELDS = []
+
+# ISO-639 code
+# for example: de, fr, ja, ko, zh-hans
+LANGUAGE_CODE = "en, tr, sq, mk"
```

Comment:
Encode to use the following languages: English, Turkish, Albanian, and Macedonian.